plankton/core/control_server.py

```diff
@@ -193,16 +196,37 @@ def __init__(self, object_map=None, host='127.0.0.1', port='10000'):
         self.port = port
```

Comment:
The `RuntimeError` above this is probably also of the category of potentially user facing errors.